JS-Algorithms/challenges.js

```diff
@@ -42,12 +50,19 @@ const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
   // YOUR CODE HERE...
+  index = index % COUNTRY_NAMES.length; //adjust the the range of the index
+  let firstPart = COUNTRY_NAMES.slice(index); //get the first part of the array
+  let secondPart = COUNTRY_NAMES.slice(0, index); //get the second part of the array to pass at the beginning
+  // console.log(firstPart)
+  // console.log(secondPart)
+  return firstPart.concat(secondPart); //just concatenate both parts of the array 
```

Comment:
It is not necessary to store the part since you're not going to use them in other parts of the code, use array spreading instead

```suggestion
   return [...COUNTRY_NAMES.slice(index), ...COUNTRY_NAMES.slice(0, index)];
```